tests/pfcwd/conftest.py

```diff
@@ -244,3 +244,24 @@ def pfcwd_pause_service(ptfhost):
         needs_resume["garp_service"] = False
 
     logger.debug("pause_service needs_resume {}".format(needs_resume))
+
+
+@pytest.fixture(scope="function", autouse=False)
+def skip_pfcwd_higher_speeds(
```

Comment:
@rraghav-cisco  For now, let's only skip the >100G ports on Cisco chassis

Reply:
If other platform also complains for this, they can find this code and skip on demand

Reply:
@yejianquan : Done.

Reply:
@yejianquan : this issue should apply to T0/T1 also so I don;t see reason for having constrain only for T2/moduar chassis.

@alpeshspatel : Please confirm. 

I would like the change to be as generic as possible. Hard to justify fanout limitation only applies to T2

Reply:
@abdosi if we skip it on all 400G. T0/T1 may got skipped unconsciously on internal-202405, 
And we can skip it because Ixia is covering most of them, but not all T0/T1 are confirmed on this.
Combined we didn't hear complaints from T0/T1, don't think skip it as generic is a better choice.

At least start only from Cisco chassis is not harmful? It's a subset of general skip.
Release manager could also challenge on that if we skip it generally.